inlong-dashboard/src/plugins/groups/common/GroupDefaultInfo.ts

```diff
@@ -39,12 +39,12 @@ export class GroupDefaultInfo implements DataWithBackend, RenderRow, RenderList
   @FieldDecorator({
     type: 'input',
     props: {
-      maxLength: 100,
+      maxLength: 200,
     },
     rules: [
       { required: true },
       {
-        pattern: /^[a-z_0-9]+$/,
+        pattern: /^[a-zA-Z0-9_.-]+$/,
         message: i18n.t('meta.Group.InlongGroupIdRules'),
```

Comment:
Please update the description of `meta.Group.InlongGroupIdRules` at the same time.

Reply:
Done.